docs/getting_started/installation_and_usage.md

```diff
@@ -175,6 +175,15 @@ tools.build:compiler_executables={"c": "gcc-12", "cpp": "g++-12"}
     [as system headers support]: https://github.com/mpusz/mp-units/releases/tag/v2.0.0
 
 
+[`MP_UNITS_BUILD_CXX_MODULES`](#MP_UNITS_BUILD_CXX_MODULES){ #MP_UNITS_BUILD_CXX_MODULES }
+
+:   [:octicons-tag-24: 2.2.0][build_cxx_modules support] · :octicons-milestone-24: `ON`/`OFF` (Default: `OFF`)
+
+    Add C++ modules to the list of default targets.
```

Comment:
I do not think it is correct. It says what it does, but now what is the outcome. From this description, it is not obvious that all examples and tests are built using modules.